flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -135,7 +135,7 @@ type embeddedFlow struct {
 	Transport    *flow.TransportLayer `json:"Transport,omitempty"`
 	ICMP         *flow.ICMPLayer      `json:"ICMP,omitempty"`
 	DHCPv4       *fl.DHCPv4           `json:"DHCPv4,omitempty"`
-	DNS          *fl.DNS              `json:"DNS,omitempty"`
+	DNS          []*fl.DNS            `json:"DNS,omitempty"`
```

Comment:
as here you are changing the schema, you need to bump the version
https://github.com/skydive-project/skydive/blob/master/storage/elasticsearch/client.go#L42

Reply:
So all that needs to be done is just to change the number?

Reply:
yes just increasing the version is enough

Reply:
done